.travis/travis_before_install.sh

```diff
@@ -7,7 +7,7 @@ else
     rm -rf $HOME/miniconda2
     mkdir -p $HOME/download
     if [[ -d $HOME/download/miniconda.sh ]] ; then rm -rf $HOME/download/miniconda.sh ; fi
-    wget -c https://repo.continuum.io/miniconda/Miniconda2-4.1.11-Linux-x86_64.sh -O $HOME/download/miniconda.sh
+    wget -c https://repo.continuum.io/miniconda/Miniconda3-4.5.11-Linux-x86.sh -O $HOME/download/miniconda.sh
```

Comment:
You are now using the 32 bit version of miniconda, and switching to python 3.  Are you sure this is what you want?